Games/repo/versions/18899028.json

```diff
@@ -0,0 +1,26 @@
+{
+  "buildId": 17560228,
+  "timeUpdated": 1750171969,
+  "gameVersion": "0.1.2.51_beta",
```

Comment:
```suggestion
  "gameVersion": "0.1.2-beta.51",
```

Reply:
I have to adhere to the version format which NuGet uses, which is close to SemVer. Game developers failing to use a standard versioning system is the only thing blocking this project from being fully automated 😭